tracer.go

```diff
@@ -0,0 +1,166 @@
+package tracer
+
+// TODO: change package
```

Comment:
Maybe just make it `package wavefront`?

Reply:
I moved all file to a trace directory, similar to the wavefront-sdk-go/senders, `package wavefront` looks to generic